src/local/docker-compose.yml

```diff
@@ -118,8 +138,7 @@ services:
       - 3004:3000
     environment:
       POLYMESH_NODE_URL: 'ws://alice:9944'
-      POLYMESH_MIDDLEWARE_URL: 'http://tooling:3000'
-      POLYMESH_MIDDLEWARE_API_KEY: '${TOOLING_API_KEY}'
+      POLYMESH_MIDDLEWARE_V2_URL: 'http://graphql:3000'
```

Comment:
Should we remove the `POLYMESH_MIDDLEWARE_URL` altogether or for now just comment out and add in comment for deprecated?

Reply:
This internal to the user, and V2 is supposed to be a 100% backwards compatible replacement. I don't want to mention it because people shouldn't use it.

Also, the user can't really edit this file unless they start using `which polymesh-local` and start digging into global node_modules.

I still serve tooling like always, so any external tool using V1 will still work.